### PR TITLE
docs(popover): add onShown & onHidden output demo

### LINKED ISSUE #3807: docs(popover): add onShown & onHidden output demo
Add demo for showing and hiding events 

related to #3255

### PR DESCRIPTION
**Pull request by artolshansky**
Add demo for showing and hiding events

close #3807

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+popover/popover-section.list.ts [hunk 1]
```diff
@@ -91,6 +92,13 @@ export const demoComponentContent: ContentSection[] = [
         (like our input groups, button groups, etc) or inside elements with <code>overflow: hidden</code></p>`,
         outlet: DemoPopoverContainerComponent
       },
+      {
+        title: 'Showing/Hiding events listeners',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+popover/popover-section.list.ts]: just `Visibility events`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+popover/popover-section.list.ts (hunk 1 above)
Comment (artolshansky) [reply demo/src/app/components/+popover/popover-section.list.ts]: @IlyaSurmay thx, couldn't find a good title

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Pull request merged**